COPYRIGHT

```diff
@@ -0,0 +1 @@
+Copyright 2023-2024 Ably Real-time Ltd (ably.com)
```

Comment:
Realtime?

Reply:
Copied this from the example, but good spot!